README.md

```diff
@@ -10,6 +10,8 @@ Nextcloud app that replicates basic gpodder.net api to sync podcast consumer app
 ### Installation
 Either from the official Nextcloud app store ([link to app page](https://apps.nextcloud.com/apps/gpoddersync)) or by downloading the [latest release](https://github.com/thrillfall/nextcloud-gpodder/releases/latest) and extracting it into your Nextcloud apps/ directory.
 
+💡There is no app settings icon since we have no settings to set.
```

Comment:
Maybe something like "[...] since there are no settings that can be set."? The 'we' sounds a bit out of place there.
And is "app settings icon" intended or did you mean "app/settings icon"?
I like the symbol as an indicator.

Do we need to clarify that you can authenticate using the Nextcloud account system or is this clear?

Reply:
Thanks for the input. I adapted

Reply:
> Do we need to clarify that you can authenticate using the Nextcloud account system or is this clear?

I think it us clear. How else would you authenticate, right?

Reply:
Yeah, you're probably right, this should be self-explanatory.
The other changes are good, ready to be merged.